src/main/webapp/app/exercises/shared/structured-grading-criterion/structured-grading-criterion.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ArtemisMarkdownEditorModule } from 'app/shared/markdown-editor/markdown-editor.module';
+import { GradingInstructionsDetailsComponent } from 'app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.ts';
+
+@NgModule({
+    declarations: [GradingInstructionsDetailsComponent],
+    exports: [GradingInstructionsDetailsComponent],
+    imports: [CommonModule, ArtemisSharedModule, ArtemisMarkdownEditorModule],
+})
+export class StructuredGradingCriterionModule {}
```

Comment:
Same like the slide toggle, I could declare the component in the shared module, but I would leave it like this for now if it is ok for you, because more components might be added in the future if we extended this feature further, for example, if we extended it with self- and peer review.